guides/INSTALLING.md

```diff
@@ -19,7 +19,7 @@ In the below, the installation of each app is typically one of:
 
 Software | Install | Notes
 -------- | ------- | ----- |
-[Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/) | `$ brew install --cask adoptopenjdk8`
+[Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/) | `$ brew install openjdk@8 ` | `$ export PATH="/usr/local/opt/openjdk@8/bin:$PATH"` and `$ export JAVA_HOME="/Library/Java/JavaVirtualMachines/openjdk-8.jdk/Contents/Home/"`
```

Comment:
I'd clarify the notes to detail where to add the PATH and JAVA_HOME env vars (similar to the Node notes further down).
```suggestion
[Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/) | `$ brew install openjdk@8 ` | Append `export PATH="/usr/local/opt/openjdk@8/bin:$PATH"` and `export JAVA_HOME="/Library/Java/JavaVirtualMachines/openjdk-8.jdk/Contents/Home/"` to your shell profile (eg. `.zshrc`) and restart your terminal.
```